cli/src/query.rs

```diff
@@ -19,14 +19,15 @@ use crate::{
 #[allow(clippy::too_many_arguments)]
 pub fn query_files_at_paths(
     language: &Language,
-    paths: Vec<String>,
+    paths: &Vec<String>,
```

Comment:
If we're not mutating or consuming the Vec, we can just pass in a slice instead

```suggestion
    paths: &[String],
```